.travis.yml

```diff
@@ -44,3 +44,7 @@ before_deploy:
 #    repo: Code-4-Community/backend-scaffold
 #    branch: master
 #  skip_cleanup: true
+
+after_success:
```

Comment:
Will this run on individual PRs or only on deploys? Where does `after_success` life in the lifecycle? Could this be part of `install`? 

Unrelated to this PR entirely, but I'm not actually too sure that we're running tests on each PR, generally.

Reply:
after-success is within the script phase so it will be invoked after every successful build. Based on coveralls page, the dashboard with only work when Travis is building successfully.

https://docs.travis-ci.com/user/job-lifecycle/ This link does a good job of explaining the lifecycle.